resources/compass/charts/gateway/values.yaml

```diff
@@ -9,5 +9,11 @@ oathkeeper:
   host: ory-oathkeeper-proxy.kyma-system.svc.cluster.local
   port: 4455
 
+oathkeeper:
+  host: ory-oathkeeper-proxy.kyma-system.svc.cluster.local
```

Comment:
`kyma-system` shouldn't be hardcoded, there should be `Release.namespace` inserted, right?

Reply:
`Release.namespace` won't work, as ory components are installed as a part of ory chart, which in Kyma is installed in `kyma-system` namespace.